server/app.py

```diff
@@ -1,13 +1,85 @@
-from flask import Flask
+from flask import Flask, request, redirect, url_for, render_template, jsonify
+from flask_sqlalchemy import SQLAlchemy
 from api.ping_handler import ping_handler
 from api.home_handler import home_handler
-from models import db
+from config import DB_USERNAME, DB_PASSWORD, DB_NAME
+from flask_bcrypt import Bcrypt
+from email_validator import validate_email
+import jwt
+import json
+import re
 
 
 app = Flask(__name__)
 app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql://{}:{}@localhost/{}'.format(DB_USERNAME, DB_PASSWORD, DB_NAME)
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
-db.init_app(app)
+app.config['SECRET_KEY'] = 'bubblegum'
+db= SQLAlchemy(app)
+bcrypt = Bcrypt(app)
+
+class User(db.Model):
```

Comment:
I think you should put the User model in models.py and also I think there should be a reference to the receipts the user has created.

Reply:
I was having an issue trying to separate the model from the app.py and will look into it.
The user model should have a reference to the receipts model?